tutor4points/core/templates/tutors.html

```diff
@@ -42,9 +44,43 @@
 
 
     <div>
-        <button type ="submit" name="request" value="request" class="request-button">Request</button>
+        <button type ="submit" name="request" value="request" class="request-button" data-toggle= "modal" data-target="#tutor-request-modal">Request</button>
+    </div>
+
+    <!-- insert request form here -->
+    <div class = "modal fade" id = "tutor-request-modal">
+        <div class = "modal-dialog model-lg">
+            <div class= "modal-content">
+               <div class = "modal-flex-container">
+                   <div class = "form-container">
+                       <h2>Request {{tutor.first_name}} {{tutor.last_name}}</h2>
+                        <form method="POST" class="form-group" enctype='multipart/form-data'>
+                            {% csrf_token %}
+                            <input type="hidden" name="requestedTutor" id="requestedTutor" value="">
+                            
+                            <div class="form-row">
```

Comment:
You can do {{form | cripsy}} rather than listing each field out if you don't need to style each part of the form. (just another stylistic choice, not required)